lib/inputs/redis-increments.js

```diff
@@ -14,11 +14,10 @@ module.exports = class RedisIncrements {
   }
 
   check(record) {
-    if (record.feederPodcast || record.feederEpisode) {
-      return record.type === 'combined' && !!record.download && !record.bytesCompliance && !record.download.isDuplicate;
-    } else {
-      return false;
-    }
+    const isType = record.type === 'combined' || record.type === 'postbytes';
+    const isFeeder = record.feederPodcast || record.feederEpisode;
```

Comment:
yeah, i'm also starting to wonder about the complexity of continuing to support that.  maybe we ditch it in Dovetail 3.0

Reply:
there are products that do just that. Also probably worth thinking about supporting audio + metadata timings rather than always segmented files, but also a DT 3.0 (or 2.later) concern I think